content/blog/announcing-http-add-on.md

```diff
@@ -0,0 +1,145 @@
++++
+title = "The KEDA HTTP Add-on Beta"
+date = 2021-05-25
```

Comment:
Let's put it to next week
```suggestion
date = 2021-06-02
```

Reply:
How about I update this right before we're ready to merge.